### DISCUSSION TITLE
Plans for the future

### DISCUSSION THREAD

**Discussion by ExperiencersInternational** [ExperiencersInternational/tvsetup#10] [Announcements]
For starters, I just want to thank @dp247 for all the work they've contributed to this. In this post, I want to outline the plans that I have for the future of this repository.

First of all, I eventually want to change every icon to the white ones we use for BBC One, ITV1 etc so that it's more consistent with a Freeview Play EPG. I also want to eventually finish all of the BBC Radio channels. The main reason for me not doing them is because there's quite a few that I need to find links, icons for etc

**Comment by dp247** [comment]
![image](https://user-images.githubusercontent.com/9065463/232611052-7bd97bde-fdf8-4b66-ab04-3fd92683338f.png)
on it

**Comment by ExperiencersInternational** [comment]
I beat you to that one lol @dp247

**Comment by dp247** [comment]
Damn lol
